### PR TITLE
[CI:DOCS] Add note on debugging quadlet unit files

### PR DESCRIPTION
**Pull request by IceWreck**
#### Does this PR introduce a user-facing change?

```release-note
None
```

Wasn't mentioned in the docs anywhere and took me a while to find it.

### CODE REVIEW

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -69,6 +69,18 @@ Currently, only the `Alias`, `WantedBy` and `RequiredBy` keys are supported.
 words `WantedBy=other.service`, not `WantedBy=other.container`. The same is
 true for other kinds of dependencies, too, like `After=other.service`.
 
+### Debugging unit files
+
+After placing the unit file in a unit search path (mentioned above), you can enable it with `systemctl enable`.
```
Comment (ygalblum) [inline docs/source/markdown/podman-systemd.unit.5.md:74]: This comment is incorrect. `systemctl enable` will always fail. The reason for the failure is that the `.service` file is generated by Quadlet and as a result considered by systmed as transient which systemd does not allow enabling.

We have opened a ticket with systemd (https://github.com/systemd/systemd/issues/28006) in order to find a way to allow enabling Quadlet generated services, but have yet to receive any response

Comment (IceWreck) [reply docs/source/markdown/podman-systemd.unit.5.md]: Thank you. Fixed.

I was already using the `[Install]` section, just messed up while writing the doc.

**Review by ygalblum — APPROVED** [review]
LGTM

**Comment by ygalblum** [comment]
@containers/podman-maintainers PTAL

**Review by vrothberg — APPROVED** [review]
/lgtm

Thank you!

**Pull request merged**